apps/dlq/lib/dlq/server.ex

```diff
@@ -0,0 +1,57 @@
+defmodule Dlq.Server do
```

Comment:
Are these DLQ changes just a rename, or does it go deeper than that?

Reply:
It goes deeper, while brainstorming around source/destination, we wanted to move ownership of Json encoding/decoding the data into the source and destination implementations.  That meant that these implementations needed to DLQ.  We may refactor this again after learning more as we implement source and destinations.